test/chainrules.jl

```diff
@@ -268,10 +270,15 @@ end
         # broken because Zygoye compresses `(NoTangent(), NoTangent())` into just NoTangent()
         # which ChainRulesTestUtils does not think is valid:
         @test_broken(rrule_via_ad(ZygoteRuleConfig(), round, 2.2) isa Tuple{NoTangent,NoTangent})
```

Comment:
The comment would be correct if
```suggestion
        # broken because Zygoye compresses `(NoTangent(), NoTangent())` into just NoTangent()
        # which ChainRulesTestUtils does not think is valid:
        @test_broken(rrule_via_ad(ZygoteRuleConfig(), round, 2.2)[2](1) isa Tuple{NoTangent,NoTangent})
```
but the rrule should return a `y, pb`.

In fact it looks fine below, though the rule should probably return `ZeroTangent()` instead. (Not `NoTangent()` since we definitely can perturb the input)

Reply:
OK, I've deleted it. I was just a bit puzzled by whether this was broken at some point.